Docker/README-multiple-nodes.md

```diff
@@ -0,0 +1,22 @@
+
+# How to run more than the default 4 nodes with docker compose.
+
+The simple steps are:
+
+```
+./gen.sh 6
```

Comment:
I would probably clarify that `6` here is the number of primary nodes that will be created and that always `1 worker` will be created for each primary. Also outline that at the moment we don't offer a way to define the number of workers per primary.